src/cli.ts

```diff
@@ -21,7 +21,16 @@ export function run(): Promise<void> {
       type: Boolean,
     });
     var cli = commandLineArgs(argsWithHelp);
-    var cliOptions = cli.parse();
+
+    try {
+      var cliOptions = cli.parse();
```

Comment:
`polyserve --bla value1` isn't causing any exceptions w/o this patch. Can you add a failing test first?